torch/cuda/_dynamo_graphs.py

```diff
@@ -0,0 +1,160 @@
+import torch
+from torch.fx import GraphModule
+from torch.nn import Module
+from torch.fx.passes.backends.cudagraphs import partition_cudagraphs
+from torch.multiprocessing.reductions import StorageWeakRef
+from torch.utils._pytree import tree_map
+import torchdynamo  # type: ignore[import]
+from torchdynamo.optimizations.training import AOTAutogradStrategy  # type: ignore[import]
+
+import operator
+from collections import defaultdict
+from typing import Set
+
+# TODO: maybe this should live in torchdynamo instead
```

Comment:
noob comment: It does seem kinda cool that you can register `AOTAutogradStrategy` without changing `dynamo` or `pytorch` code